README.md

```diff
@@ -1,6 +1,6 @@
 # gocqlx [![GoDoc](http://img.shields.io/badge/go-documentation-blue.svg?style=flat-square)](http://godoc.org/github.com/scylladb/gocqlx) [![Go Report Card](https://goreportcard.com/badge/github.com/scylladb/gocqlx)](https://goreportcard.com/report/github.com/scylladb/gocqlx) [![Build Status](https://travis-ci.org/scylladb/gocqlx.svg?branch=master)](https://travis-ci.org/scylladb/gocqlx)
 
-Package `gocqlx` is a `gocql` extension, similar to what `sqlx` is to `database/sql`.
+Package `gocqlx` is a Scylla / Cassandra productivity toolkit backed by `gocql`.
```

Comment:
I think the reference to `sqlx` is relevant and important detail for people trying to figure what this package is about.